packages/suite/src/constants/suite/menu.ts

```diff
@@ -0,0 +1,13 @@
+export const MAIN_MENU_ITEMS = [
+    { text: 'Dashboard', icon: 'DASHBOARD', route: 'dashboard-index' },
+    { text: 'Wallet', icon: 'WALLET', route: 'wallet-index' },
+    { text: 'Passwords', icon: 'PASSWORDS', route: 'passwords-index' },
+    { text: 'Exchange', icon: 'EXCHANGE', route: 'exchange-index' },
```

Comment:
create empty placeholders for Passwords and Exchange in both web and desktop (now is 404)